src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
 }
+
+header {
```

Comment:
Use only classes selector, fix it everywhere
```suggestion
.header {
```

Reply:
Not fixed 
<img width="851" alt="image" src="https://github.com/user-attachments/assets/f941a0fe-be5b-46d6-a905-f288627df289">